packages/arb-validator-core/gobridge/executionChallenge.go

```diff
@@ -124,14 +126,14 @@ func (c *ExecutionChallenge) BisectAssertion(
 	c.commitToSegment(hashes)
 	c.asserterResponded()
 
-	c.client.pubMsg(c.challengeData, arbbridge.MaybeEvent{
+	c.client.pubMsg(c.challenge, arbbridge.MaybeEvent{
 		Event: arbbridge.ExecutionBisectionEvent{
 			ChainInfo: arbbridge.ChainInfo{
 				BlockId: c.client.getCurrentBlock(),
 			},
 			Assertions: assertions,
 			TotalSteps: totalSteps,
-			Deadline:   c.client.challenges[c.contractAddress].deadline,
+			Deadline:   c.client.challenges[c.contractAddress].challengeData.deadline,
```

Comment:
Unnecessary indirection to get `challengeData`

Reply:
fixed